### PR TITLE
Button Component Refactor

### PR DESCRIPTION
**Pull request by bharati-21**
Refactored button component
- Used margin and padding classes
- Changed accent to secondary color
- Edited code snippets
- Edited button links
- Moved button component to a separate file docs/button/index.html from docs/index.html

### CODE REVIEW

File: Components/buttons/button.css [hunk 1]
```diff
@@ -1,170 +1,189 @@
 .btn {
-    width: max-content;
+    /* Size properties */
     display: inline-block;
-    padding: var(--space-0-5) var(--space-0-75);
+    vertical-align: middle;
+    width: max-content;
     min-width: 80px;
```
Comment (rishbitsnbytes) [inline Components/buttons/button.css:6]: rems maye a better choice here. Also will be consistent.

File: Components/buttons/button.css [hunk 2]
```diff
@@ -1,170 +1,189 @@
...
     -ms-border-radius: var(--br-50);
     -o-border-radius: var(--br-50);
+    border: none;
+
     width: var(--space-3);
     height: var(--space-3);
     min-width: var(--space-3);
     min-height: var(--space-3);
+
     display: flex;
```
Comment (rishbitsnbytes) [inline Components/buttons/button.css:158]: Spacings ? If it's for the sake of visual sectioning for reviewers then it's okay!

**Review by rishbitsnbytes — COMMENTED** [review]
Just minor comments. Variable usage looks great. Comments are also good. Looks good to merge.

File: Components/buttons/button.css (hunk 1 above)
Comment (bharati-21) [reply Components/buttons/button.css:6]: Hey, so var(--space-0-5) is 0.5rem here. 
Or are you referring to the width?

File: Components/buttons/button.css (hunk 2 above)
Comment (bharati-21) [reply Components/buttons/button.css:158]: Yes, it is for that purpose. I have sectioned properties together. E.g. display, font, flex, size, border, etc. Makes it easier to debug, add, and review.

**Comment by bharati-21** [comment]
> Just minor comments. Variable usage looks great. Comments are also good. Looks good to merge.

Thank you!

**Review by shashikantmaurya0007 — APPROVED** [review]
Looks good to merge

**Comment by bharati-21** [comment]
> Looks good to merge

Thank you!

File: Components/buttons/button.css (hunk 1 above)
Comment (rishbitsnbytes) [reply Components/buttons/button.css:6]: I meant that for min-width property.

Comment (bharati-21) [reply Components/buttons/button.css:6]: Okay i shall change it to rems. Thank you.

**Pull request merged**